flink-table/flink-table-common/src/main/java/org/apache/flink/table/functions/BuiltInFunctionDefinitions.java

```diff
@@ -674,6 +674,14 @@
 			.outputTypeStrategy(TypeStrategies.MISSING)
 			.build();
 
+	// json functions
+	public static final BuiltInFunctionDefinition JSON_EXISTS =
+		new BuiltInFunctionDefinition.Builder()
+			.name("json_exists")
+			.kind(SCALAR)
+			.outputTypeStrategy(TypeStrategies.MISSING)
```

Comment:
It's weird, why not directly specify the outputType? I know that all function definitions are all defined with `MISSING` strategy, and it make me more confuse.

Reply:
> It's weird, why not directly specify the outputType? I know that all function definitions are all defined with `MISSING` strategy, and it make me more confuse.

The type specified here becomes a cast.